Mods/dopp_ReAerah.theme

```diff
@@ -0,0 +1,25 @@
+@description ReAerah
+@author dopp
+@version 18
+@provides ReAerah.ReaperThemeZip https://github.com/doppgit/theme/raw/master/ReAerah.ReaperThemeZip
```

Comment:
Thanks for uploading your theme here! A small change to the package file is required before I can merge this:

```suggestion
@provides ReAerah.ReaperThemeZip https://github.com/doppgit/theme/raw/61f56011d04e54454be06cd6db21f07575f2ba02/ReAerah.ReaperThemeZip
```

Currently, "master" in the download URL always points to the latest version of the file. The URL should instead be unique to version 18 and permanent. This can be done by replacing "master" with version 18's commit hash like done above.

For each future updates of the theme, both the package's version number and the download URL should be be edited.